robocopkaka/event-manager-rails
# 163369404 Admins can add centers (UI)

#### What does this PR do?
Give admins the ability to add new centres
#### Description of Task to be completed?
* Provide a form that admins to add centres with
* Enable image upload
#### How should this be manually tested?
* Visit `http://{baseURL}/add-center`
* Fill in the form and select an image to be uploaded
* Click submit
* You should see a toast message telling you that your request was successful and you should be redirected to the page showing all centres
#### Any background context you want to provide?
You won't see the new centre when you're redirected to the page showing all centres till you've triggered a page refresh. This will be taken care off in a different PR
#### What are the relevant pivotal tracker stories? (if applicable)
[#163369404](https://www.pivotaltracker.com/story/show/163369404)
